src/pages/tags/[...slug].astro

```diff
@@ -58,6 +58,6 @@ const { Content } = await entry.render();
             )
         }
 
-        <Content components={{ Aside: Aside }} />
+        <Content components={{ aside: aside }} />
```

Comment:
Revert to `components={{ Aside: Aside }}` to reflect the import.

Love this by the way.